### ISSUE TITLE
Create a tool to visually digest this data easily

### ISSUE THREAD

**Issue by krafts** [hypergig/cafe-devops#4]

**Comment by krafts** [comment]
Perhaps somekind of a tabular format. And definitely something that aggregates all the ratings into overall values (#5 added (will be adding) multiple ratings).

**Comment by hypergig** [comment]
Agreed. Was waiting for more data first, I have a feeling our standards will change a great deal in the beginning.  Reinvestigate after we stabilize.
